src/components/MyAccount/RequestsTab/CancelButton.tsx

```diff
@@ -0,0 +1,118 @@
+// @ts-nocheck
+// Modal onClose
+import { useState } from "react"
+import type { Hold, Patron } from "../../../types/accountTypes"
+import {
+  useModal,
+  Box,
+  Icon,
+  Button,
+  Text,
+  Heading,
+} from "@nypl/design-system-react-components"
+import { BASE_URL } from "../../../config/constants"
+import styles from "../../../../styles/components/MyAccount.module.scss"
+
+const CancelButton = ({
+  removeHold,
+  hold,
+  patron,
+}: {
+  removeHold: (hold: Hold) => void
+  hold: Hold
+  patron: Patron
+}) => {
+  const { onOpen: openModal, onClose: closeModal, Modal } = useModal()
+
+  function confirmModalProps(hold) {
+    return {
+      bodyContent: (
+        <Box className={styles.modalBody}>
+          <Text sx={{ marginLeft: "l", marginRight: "m" }}>
+            Your request for{" "}
+            <span style={{ fontWeight: 510 }}>{hold.title}</span> has been
```

Comment:
Normally font weights are multiples of 100, but i think a DS variable can be used instead of a number here.